datasets/so_stacksample/so_stacksample.py

```
@@ -0,0 +1,173 @@
+        `datasets.load_dataset("so_stacksample", "Answers", data_dir="/home/<user>/Downloads")`
+        will load the `Answers.csv` dataset.
+        """
+
+    def _info(self):
+        if self.config.name == "Answers":  # This is the name of the configuration selected in BUILDER_CONFIGS above
+            features = datasets.Features(
+                {
+                    "Id": datasets.Value("int32"),
+                    "OwnerUserId": datasets.Value("int32"),
+                    "CreationDate": datasets.Value("string"),
+                    "ParentId": datasets.Value("int32"),
+                    "Score": datasets.Value("int32"),
+                    "Body": datasets.Value("string"),
+                }
+            )
+        elif self.config.name == "Questions":
+            features = datasets.Features(
+                {
+                    "Id": datasets.Value("int32"),
+                    "OwnerUserId": datasets.Value("int32"),
+                    "CreationDate": datasets.Value("string"),
+                    "ClosedDate": datasets.Value("string"),
+                    "Score": datasets.Value("int32"),
+                    "Title": datasets.Value("string"),
+                    "Body": datasets.Value("string"),
+                }
+            )
+        else:
+            features = datasets.Features(
+                {
+                    "Id": datasets.Value("int32"),
+                    "Tag": datasets.Value("string"),
+                }
+            )
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            supervised_keys=None,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+        path_to_manual_file = os.path.join(
+            os.path.abspath(os.path.expanduser(dl_manager.manual_dir)), self.config.name + ".csv"
+        )
```

can you raise an error here if the file doesn't exist ?

You can find an example here : https://github.com/huggingface/datasets/blob/master/datasets/arxiv_dataset/arxiv_dataset.py#L104-L109